src/app/app.component.html

```diff
@@ -81,6 +81,10 @@
 
     <mat-divider></mat-divider>
 
+    <app-saved-searches (searchClicked)="searchFor($event)"></app-saved-searches>
```

Comment:
This is the bit that reopens the search? Would it be possible to add more magic, so that if the search was saved with the extra search options (wrench) menu open, it opens + repopulates it?

Reply:
The fancy wrench options just modify the search query, so we do save it with everything in there. We could make it so that when a query is intered the wrench options automagically pick up what's in there, but that'll require pretty much reimplementing the xapian query parser :/

Reply:
right, but we cant reload it and then edit it.. for eg just wanting to change the date on a saved one.. hmm could just also store a json rep of the fields in the advanced search?

Reply:
Storing JSON would work until someone modifies the query in the wrench UI and then adds some search terms in the regular search bar – already possible, and the changes are not reflected in the wrench UI then.

Sounds like a bigger task than what we have here – including the fact that our issue here is basically the symptom of the wrench UI not updating automatically after search queries are entered manually. I'll open a separate issue for this, as this will likely require exposing Xapian's query parser in our JS code if we wanna do it right (or plenty of regexes if we want to make it good enough).

Reply:
Moved to https://github.com/runbox/runbox7/issues/755 which links here, so resolving this for now :)